contracts/test/v0.8/automation/UpkeepTranscoder.test.ts

```diff
@@ -31,10 +31,7 @@ describe('UpkeepTranscoder', () => {
     const encodedData = '0xc0ffee'
 
     it('reverts if the from type is not an enum value', async () => {
-      await evmRevert(
```

Comment:
@smartcontractkit/keepers this test failed after bumping Hardhat as it didn't revert with the given message. It still reverts so I removed the message component

Reply:
I'm not worried about it